src/DaPigGuy/PiggyCustomEnchants/enchants/armor/EnlightedEnchant.php

```diff
@@ -23,14 +23,6 @@ class EnlightedEnchant extends ReactiveEnchantment
     /** @var string */
     public $name = "Enlighted";
 
-    /**
-     * @return array
-     */
-    public function getReagent(): array
-    {
-        return [EntityDamageByEntityEvent::class];
-    }
-
```

Comment:
This class should ``use ReactiveTrait`` instead of extending ReactiveEnchantment
> Couldn't comment line 21, so I commented this line